docs/cumulusci_flow.rst

```diff
@@ -78,12 +98,60 @@ If this combination of named parent and child branches exist, the auto-merging f
 * Parent branches do receive the merge from main which kicks off a Feature Test build
 * At the end of a successful Feature Test build on a Parent branch, the parent branch is auto-merged into all child branches
 
-This allows us to support multiple developers working on one big feature while keeping the whole feature isolated from main until we're ready to release it.  The parent branch is the branch representing the overall feature.  Each developer can create child branches for individual components of the larger feature.  Their child branch still gets CI builds like all feature branches.  When they are ready to merge from their child branch to the parent branch, they create a Pull Request which gets code reviewed by other developers working on the parent feature branch and finally merged to the parent branch.
+This allows us to support multiple developers working on a single large feature while keeping that feature isolated from main until we're ready to release it.  The parent branch is the branch representing the overall feature.  Each developer can create child branches for individual components of the larger feature.  Their child branch still gets CI builds like all feature branches.  When they are ready to merge from their child branch to the parent branch, they create a Pull Request which gets code reviewed by other developers working on the parent feature branch and finally merged to the parent branch.
+
+CumulusCI facilitates parent to child auto-merges via the `github_parent_to_children` task, which is included by deault in the `ci_feature` flow.  If a parent feature branch passes the build, it is automatically merged into all child branches.
+
+Release Branches
+----------------
+Some teams deliver large releases several times a year.
+For this type of release cadence, Salesforce.org uses a special type of branch referred to as a release branch. These long-lived branches are created off of the ``main`` branch, serve as the target branch for all features associated with that release and are eventually merged back to the ``main`` branch when a release occurs.
+To be able to clearly track what work is associated with a specific release, release branches adhere to the following:
+   * They are the parent branches of ALL feature work associated with a release. Put another way; all feature branches use the parent-child naming convention with its target release branch.
+   * Use a strict naming format: ``feature/release_num`` where ``release_num`` is a valid integer.
```

Comment:
Have you checked the rendered docs? With reStructuredText bulleted lists should usually have a blank line before them and no indent, or else they'll be rendered with an extra level of indentation.

Reply:
Have definitely checked rendered docs, but may have not noticed this. Checking again.

Reply:
It may vary depending on what system is rendering it. It's not a big deal.